overlord/snapstate/snapstate.go

```diff
@@ -302,8 +306,68 @@ func Remove(s *state.State, name string) (*state.TaskSet, error) {
 
 // Rollback returns a set of tasks for rolling back a snap.
 // Note that the state must be locked by the caller.
-func Rollback(s *state.State, snap, ver string) (*state.TaskSet, error) {
-	return nil, fmt.Errorf("rollback not implemented")
+func Rollback(s *state.State, name, ver string) (*state.TaskSet, error) {
```

Comment:
bit confused about what should happen when ones rollback again? should we stop that here?